disperser/apiserver/server_test.go

```diff
@@ -277,6 +279,85 @@ func TestDisperseBlobWithExceedSizeLimit(t *testing.T) {
 	assert.Equal(t, err.Error(), "blob size cannot exceed 2 MiB")
 }
 
+func TestRatelimit(t *testing.T) {
+	data50KiB := make([]byte, 50*1024)
+	_, err := rand.Read(data50KiB)
+	assert.NoError(t, err)
+	data1KiB := make([]byte, 1024)
+	_, err = rand.Read(data1KiB)
+	assert.NoError(t, err)
+
+	// Try with a non-allowlisted IP
+	p := &peer.Peer{
+		Addr: &net.TCPAddr{
+			IP:   net.ParseIP("0.0.0.0"),
+			Port: 51001,
+		},
+	}
+	ctx := peer.NewContext(context.Background(), p)
+
+	_, err = dispersalServer.DisperseBlob(ctx, &pb.DisperseBlobRequest{
+		Data: data50KiB,
+		SecurityParams: []*pb.SecurityParams{
+			{
+				QuorumId:           0,
+				AdversaryThreshold: 50,
+				QuorumThreshold:    100,
+			},
+		},
+	})
+	assert.ErrorContains(t, err, "account throughput limit")
+
+	for i := 0; i < 10; i++ {
```

Comment:
This should error out because blob rate (3 blobs/sec) times bucket size (3 seconds) is equal to 9 blobs, which is less than 10, right? 

Maybe add a comment?

Reply:
Yes, that's exactly what's being tested as asserted in L323 (I'm not specifying which of the 10 requests should be failing because I don't think it's 100% deterministic due to timing. Just asserts that the last request fails due to account blob limit)
I'll add a comment as well